src/CMakeLists.txt

```diff
@@ -23,26 +23,54 @@ configure_file(libtriton_dali.ldscript libtriton_dali.ldscript COPYONLY)
 
 add_subdirectory(dali_executor)
 
+add_library(
+        triton-dali-backend-utils STATIC
+        utils/triton.cc
+        config_tools/config_tools.cc
+)
+
+target_include_directories(
+        triton-dali-backend-utils PUBLIC
+        ${tritondalibackend_SOURCE_DIR}
+        ${DALI_INCLUDE_DIR}
+)
+
+target_link_libraries(
+        triton-dali-backend-utils PUBLIC
+        triton-core-serverstub
+        triton-backend-utils
+)
+
+set_target_properties(
+        triton-dali-backend-utils
+        PROPERTIES
+            POSITION_INDEPENDENT_CODE ON
```

Comment:
Do you need all these options?
Is it based on any example from the TRITON repo or it was selected based on trials and errors?

Reply:
For actual backend lib it is based on Triton example. For those helper libraries it was copied and it's actually not needed, so I removed all of the arguments except for C++17.